asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -147,14 +151,28 @@ const ExportPage = (props) => {
                       onChange={handleFile}
                     >
                       <MenuItem value="dataset" divider>
+                        <Box>
+                          <Typography variant="subtitle1">
+                            Full dataset
+                          </Typography>
+                          <Typography
+                            variant="body2"
+                            gutterBottom
+                            sx={{ color: "text.secondary" }}
+                          >
+                            With relevant and irrelevant labels
+                          </Typography>
+                        </Box>
+                      </MenuItem>
+                      <MenuItem value="dataset_relevant" divider>
                         <Box>
                           <Typography variant="subtitle1">Dataset</Typography>
```

Comment:
Thank you for your review. I think the "Relevant Subset" might be a better title. Can you please check @terrymyc Thank you.

Reply:
I think it is easy with the current setup. However, the naming convention (titles and description) at the frontend will be a crucial decision to make, because there will be five options now: All, Relevant, Irrelevant, Unlabelled, Project

Reply:
If we also want to provide the option for exporting irrelevant records, I need to redesign this interface to accommodate multiple dataset export options.

Reply:
I propose only two options:
- 1 dataset with all records
- 3 datasets with subset of the records

and not to have all separate options

Reply:
I think the 3 datasets option will have some major implications and will essentially require modifications to multiple functions in the backend. I think @J535D165 will have a better idea of how we can go about it.